ControlSettings/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+# If not stated otherwise in this file or this component's LICENSE file the
+# following copyright and licenses apply:
+#
+# Copyright 2022 Sky UK
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required (VERSION 2.6)
+
+Project(CONTROLSETTINGS-PLUGINS)
+
+set(CMAKE_BUILD_TYPE "Debug")
+
+set(PLUGIN_NAME ControlSettings)
+
+set(MODULE_NAME ${NAMESPACE}${PLUGIN_NAME})
+
+set(PLUGIN_CONTROLSETTINGS_AUTOSTART "false" CACHE STRING "Automatically start Tvmgr plugin")
+
+find_package(WPEFramework)
+find_package(${NAMESPACE}Plugins REQUIRED)
+
+set (CMAKE_CXX_STANDARD 11)
+
+find_path (STAGING_INCDIR glib-2.0)
+include_directories(${STAGING_INCDIR})
+include_directories(${STAGING_INCDIR}/glib-2.0)
+include_directories(${STAGING_INCDIR}/../lib/glib-2.0/include)
+include_directories(${STAGING_INCDIR}/rdk/iarmbus)
+include_directories(${STAGING_INCDIR}/rdk/iarmmgrs-hal)
+include_directories(${STAGING_INCDIR}/rdk/tv-hal)
+include_directories(${STAGING_INCDIR}/rdk/ds-hal)
+include_directories(${STAGING_INCDIR}/rdk/ds-rpc)
+include_directories(${STAGING_INCDIR}/rdk/ds)
+
```

Comment:
please use find_package

find_package(DS)
find_package(IARMBus)
...

like it's done in other services

Reply:
Thanks for your comments. 
Will change it in future commits.